kitsune/search/v2/search.py

```diff
@@ -114,7 +115,13 @@ def get_highlight_fields(self):
         ]
 
     def get_filter(self):
-        filters = [DSLQ("term", _index=self.get_index())]
+        # Add default filters:
+        # 1. limit scope to the Wiki index
+        # 2. exclude archived articles
```

Comment:
nit: I'd put each of these comments directly above the line which does the commented-on filtering